### PR TITLE
Triple_T_Game Project Milestone-I

### PR DESCRIPTION
**Pull request by necmigunduz**
In this project, we did:
-create the setup for Triple_T_Game project.
-add linters files,
-add setup.

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# ! /usr/bin/env ruby
```
Comment (jubaan) [inline bin/main.rb]: - [x] the expression has a space between `#` and `!`, which makes the script not work as expected. please remove the space or use the next expression `#! /usr/bin/env ruby`

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @necmigunduz and @SamArsanios. Thanks for submitting you project!

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] Please edit the PR request, it should be a brief summary or changelog of the things you have done on this step.
- [x] The shebang expression is not written correctly. see inline comments

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @SamArsanios  & @necmigunduz**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

- [x] the file is not executable. you should follow [these ](https://commandercoriander.net/blog/2013/02/16/making-a-ruby-script-executable)instructions on how to make it executable.
if you already did so, you should upload the permissions to GitHub so that anyone who clones it has the permissions. you can do that by running this command `
git update-index --chmod=+x ./bin/main.rb`.
Please double check that the shebang line is written like this `#!/usr/bin/env ruby` with the same spacing. 


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @necmigunduz  & @SamArsanios**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**